build.fsx

```diff
@@ -537,42 +536,42 @@ Target.create"CreateTag" (fun _ ->
     Git.Branches.pushTag "" "origin" release.NugetVersion
 )
 
-Target.create"Release" Target.DoNothing
+Target.create"Release" ignore
```

Comment:
Maybe we can add a space after create :)

Reply:
yup 😄